build_python_wheel.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash -e
```

Comment:
is this file intended to be used interactively? like, `configure`?

Reply:
Eh, this one seem deprecated.  scripts/make_wheel.sh is better I think?  Not sure.

Reply:
this one uses `make_wheel.sh`. I don't know if this is just for the manual interactive build.